.github/workflows/tests.yaml

```diff
@@ -107,6 +116,11 @@ jobs:
 
           python continuous_integration/scripts/host_info.py
 
+      - name: Enable queuing
+        if: ${{ matrix.queuing == 'queue' }}
+        shell: bash -l {0}
+        run: export DASK_DISTRIBUTED__SCHEDULER__WORKER_SATURATION=1.0
+
```

Comment:
I thought that environment variables are reset at every step of the github action?